README.md

```diff
@@ -117,7 +117,7 @@ This needs to be done under `CoC Options` in the Wordpress admin backend in the
 For example when running the API server locally, this could be the settings:
 ```
 Your API Key: secret
-API base url: http://localhost:1337
+API base url: http://host.docker.internal:1337
```

Comment:
:thinking: what domain is that? I don't have any domains like that available on my machine when I start `docker-compose up`. The only thing that is available is `localhost` and the exposed ports by docker.

Reply:
The url host.docker.internal can be used in Docker to resolve the IP address of the host machine (https://docs.docker.com/docker-for-windows/networking/#use-cases-and-workarounds).
As the port 1337 from the CoC API Server is exposed to the host machine, it can be accessed through it. So in the end the host ist the link between Backend and Frontend.
This works just fine for me. The feature is also available for Mac. Unfortunately I just discovered, that the feature is not OS agnostic and is not supported on Linux environments at the moment. But there is a Github issue for this at https://github.com/docker/for-linux/issues/264 and there was apparently a fix for it commited about 24h hours ago. So it could be fixed very soon.